JS/app.js

```diff
@@ -0,0 +1,45 @@
+//Sec 1
+//1. Concatenation is to combine strings. Interpolation is using a variable to represent a value.
+console.log('hello '+'world.');
+const word = "hello";
+console.log(word + " world.");
+//2.Do Not Repeat Yourself. Program runs more efficient and easier to interpret by humans. Loops.
+
+//Sec 2
+ const a = 4;
+ const b = 53;
+ const c = 57;
+ const d = 16;
+ const e = 'Kevin';
+
+console.log(a < b);
+console.log(c > d);
+console.log('Name' === 'Name');
+console.log(a < b < c);
```

Comment:
this returns true but not for the reasons you think..
what it's looking for is `a + b === c`